salt/metalk8s/orchestrate/downgrade/precheck.sls

```diff
@@ -2,8 +2,7 @@
 
 {#- When downgrading saltenv should be the newest version #}
 {%- set nodes_versions = pillar.metalk8s.nodes.values() | map(attribute='version') | list %}
-{%- do nodes_versions.sort(cmp=salt.pkg.version_cmp, reverse=True) %}
-{%- set expected = nodes_versions | first %}
+{%- set expected = salt.metalk8s.cmp_sorted(nodes_versions, cmp=salt.pkg.version_cmp, reverse=True) | first %}
```

Comment:
Is this related to Python3 compat?

Reply:
Moreover, I don't have this `cmp_sorted` method available in `salt/_modules/metalk8s.py` :confused:

Reply:
Oh shit, you right it's related to Python3 migration but not to Python3 compat, it shouldn't be in this PR. I will remove it